frontend/appflowy_flutter/packages/appflowy_ui/lib/src/theme/data/appflowy_default/semantic.dart

```diff
@@ -0,0 +1,383 @@
+      infoLightHover: AppFlowyPrimitiveTokens.blue200,
+      infoThick: AppFlowyPrimitiveTokens.blue500,
+      infoThickHover: AppFlowyPrimitiveTokens.blue600,
+      successLight: AppFlowyPrimitiveTokens.green100,
+      successLightHover: AppFlowyPrimitiveTokens.green200,
+      successThick: AppFlowyPrimitiveTokens.green600,
+      successThickHover: AppFlowyPrimitiveTokens.green700,
+      warningLight: AppFlowyPrimitiveTokens.orange100,
+      warningLightHover: AppFlowyPrimitiveTokens.orange200,
+      warningThick: AppFlowyPrimitiveTokens.orange600,
+      warningThickHover: AppFlowyPrimitiveTokens.orange700,
+      errorLight: AppFlowyPrimitiveTokens.red100,
+      errorLightHover: AppFlowyPrimitiveTokens.red200,
+      errorThick: AppFlowyPrimitiveTokens.red600,
+      errorThickHover: AppFlowyPrimitiveTokens.red700,
+      errorSelect: AppFlowyPrimitiveTokens.redAlphaRed50010,
+      purpleLight: AppFlowyPrimitiveTokens.purple100,
+      purpleLightHover: AppFlowyPrimitiveTokens.purple200,
+      purpleThickHover: AppFlowyPrimitiveTokens.purple600,
+      purpleThick: AppFlowyPrimitiveTokens.purple500,
+    );
+
+    final surfaceColorScheme = AppFlowySurfaceColorScheme(
+      primary: AppFlowyPrimitiveTokens.neutralWhite,
+      overlay: AppFlowyPrimitiveTokens.neutralAlphaBlack60,
+    );
+
+    final backgroundColorScheme = AppFlowyBackgroundColorScheme(
+      primary: AppFlowyPrimitiveTokens.neutralWhite,
+      secondary: AppFlowyPrimitiveTokens.neutral100,
+      tertiary: AppFlowyPrimitiveTokens.neutral200,
+      quaternary: AppFlowyPrimitiveTokens.neutral300,
+    );
+
+    final brandColorScheme = AppFlowyBrandColorScheme(
+      skyline: Color(0xFF00B5FF),
+      aqua: Color(0xFF00C8FF),
+      violet: Color(0xFF9327FF),
+      amethyst: Color(0xFF8427E0),
+      berry: Color(0xFFE3006D),
+      coral: Color(0xFFFB006D),
+      golden: Color(0xFFF7931E),
+      amber: Color(0xFFFFBD00),
+      lemon: Color(0xFFFFCE00),
+    );
+
+    final otherColorsColorScheme = AppFlowyOtherColorsColorScheme(
+      textHighlight: AppFlowyPrimitiveTokens.blue200,
+    );
+
+    return AppFlowyThemeData._(
+      textStyle: textStyle,
+      textColorScheme: textColorScheme,
+      borderColorScheme: borderColorScheme,
+      fillColorScheme: fillColorScheme,
+      surfaceColorScheme: surfaceColorScheme,
+      backgroundColorScheme: backgroundColorScheme,
+      iconColorScheme: iconColorScheme,
+      brandColorScheme: brandColorScheme,
+      otherColorsColorScheme: otherColorsColorScheme,
+      borderRadius: borderRadius,
+      spacing: spacing,
+      shadow: shadow,
+      brightness: Brightness.light,
+    );
+  }
+
+  factory AppFlowyThemeData.dark() {
+    final textStyle = AppFlowyBaseTextStyle();
+    final borderRadius = themeBuilder.buildBorderRadius();
+    final spacing = themeBuilder.buildSpacing();
+    final shadow = themeBuilder.buildShadow(Brightness.dark);
+
+    final textColorScheme = AppFlowyTextColorScheme(
```

Comment:
I'm wondering why we keep the border radius, spacing, and shadow builders, but remove the color scheme builders.